### PR TITLE
Adds The Pilgrim's Lunch

### PR DESCRIPTION
**Pull request by Spacedong**
Added "The Pilgrim's Lunch", a new sandwich recipe.

:cl: Spacedong
add: Added "The Pilgrim's Lunch", a new sandwich that the chef can make. It requires 2 bread slices, 1 cheese wedge, 1 steak, and 5u black pepper.
/:cl:

### CODE REVIEW

**Comment by t-f-w** [comment]
hitlers breakfast you mean? I can't remember his breakfast. His lunch tho, if i remember correctly, was usually light. A sandwich with no sauce or anything. A friend told me this but that friend is pretty knowledgeable
It's not called a pilgrims lunch i just call it that cuz it's plain, simple, and easy to carry He had what i call a Pilgrim Breakfast. A sandwich with meat and cheese, nothing else. At least I think that's what he ate. I personally am a fan of sandwiches with meat n cheese but no condiments except pepper. I make it then i press the sandwich as flat as possible

+1

**Comment by Arzion5** [comment]
Forced **MEEM**

**Comment by banthebantz** [comment]
i beg of thee, add a methmedicium reagent, have it inside the sandwich and do 
taste_description = "bricks"

**Comment by fluxcapacitor1337** [comment]
we need people to automatically say various methmedic quotes after having consumed this

**Comment by fluxcapacitor1337** [comment]
it's what the man wants himself damnit !!!!

https://i.imgur.com/IxBz4Ia.png

File: hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/obj/item/reagent_containers/food/snacks/pilgrim
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm]: Couldn't you make this a child of a regular sandwich and save yourself a few variables?

**Comment by McBawbaggings** [comment]
LOL
+1

File: hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm (hunk 1 above)
Comment (Spacedong) [reply hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm]: no

**Comment by bardofrage** [comment]
+1. Obviously.

**Comment by Spacedong** [comment]
thats 2 headmin +1's lets merge boys

**Comment by McBawbaggings** [comment]
it needs +3 due to the increase in GMs and HAs

File: hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm (hunk 1 above)
Comment (GuyonBroadway) [reply hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm]: That wasn't really a request, if you made it a child of a normal sandwich you could skip out on re-defining the sprite and plate junk.

**Comment by Spacedong** [comment]
no :rage:

**Comment by Spacedong** [comment]
fine bitch

File: hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm]: make it a child

File: hippiestation/code/modules/food_and_drinks/recipes/tablecraft/recipes_sandwich.dm [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/datum/crafting_recipe/food/pilgrim
+	name = "Pilgrim's Lunch"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/breadslice/plain = 2,
+		/obj/item/reagent_containers/food/snacks/meat/steak = 1,
+		/obj/item/reagent_containers/food/snacks/cheesewedge = 1,
+		/datum/reagent/consumable/blackpepper = 5
+	)
+	tools = list(/obj/item/kitchen/rollingpin)
+	result = /obj/item/reagent_containers/food/snacks/pilgrim
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/food_and_drinks/recipes/tablecraft/recipes_sandwich.dm]: You forgot to change the path here too dummy!

**Comment by McBawbaggings** [comment]
>commit
child

i laughed more than i should @Spacedong

**Comment by banthebantz** [comment]
/obj/item/reagent_containers/food/snacks/sandwich/pilgrim/Initialize()
    . =..()
    if(prob(50))
        name = "pilgrim's breakfast"

# make it so

**Comment by dreamcrab** [comment]
i approve +1488 as long as it doesnt make you say stuff or whatever

just a simple, light lunch

File: hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/obj/item/reagent_containers/food/snacks/sandwich/pilgrim
+	name = "pilgrim's lunch"
+	desc = "Hitler's breakfast you mean? I can't remember his breakfast. His lunch though, if I remember correctly, was usually light. A sandwich with no sauce or anything. A friend told me this but that friend is pretty knowledgeable. It's not called a pilgrims lunch I just call it that because it's plain, simple, and easy to carry."
+	tastes = list("meat" = 2, "cheese" = 1, "bread" = 2, "pepper" = 1)
+	foodtype = GRAIN
```
Comment (KayeArray) [inline hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm:5]: w_class = WEIGHT_CLASS_TINY
 it's plain, simple, and **easy to carry**

**Comment by Dudeinyourbackyard** [comment]
+1 w/o quotes
the pilgrim's brunch is very powerful

File: hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm (hunk 3 above)
Comment (Spacedong) [reply hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm]: sandwiches are already tiny you FUCK

Comment (KayeArray) [reply hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm:5]: you 100% sure?

**Comment by Spacedong** [comment]
this is ready to merge, im not adding the bullshit one-off chem or making it make you say stuff, its a light lunch ready to go

File: hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm (hunk 3 above)
Comment (Spacedong) [reply hippiestation/code/modules/food_and_drinks/food/snacks_sandwichtoast.dm]: all food items are tiny

File: hippiestation/code/modules/food_and_drinks/recipes/tablecraft/recipes_sandwich.dm [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/datum/crafting_recipe/food/pilgrim
+	name = "Pilgrim's Lunch"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/breadslice/plain = 2,
+		/obj/item/reagent_containers/food/snacks/meat/steak = 1,
+		/obj/item/reagent_containers/food/snacks/cheesewedge = 1,
+		/datum/reagent/consumable/blackpepper = 5
+	)
+	tools = list(/obj/item/kitchen/rollingpin)
```
Comment (JohnGinnane) [inline hippiestation/code/modules/food_and_drinks/recipes/tablecraft/recipes_sandwich.dm:9]: you need rolling pin to make sandwich?

Comment (KayeArray) [reply hippiestation/code/modules/food_and_drinks/recipes/tablecraft/recipes_sandwich.dm:9]: *should we tell him?*

Comment (Spacedong) [reply hippiestation/code/modules/food_and_drinks/recipes/tablecraft/recipes_sandwich.dm]: @JohnGinnane inside joke

**Comment by Arzion5** [comment]
All is lost

**Pull request merged**